public/src/actions/login.jsx

```diff
@@ -42,12 +42,11 @@ export function loginUser(loginInfo, history) {
   });
 }
 
-export function logoutUser(history) {
+export function logoutUser() {
   return dispatch => axios.get('/logout')
   .then(() => {
     localStorage.removeItem('auth_token');
     dispatch(logout());
-    history.push('/login');
```

Comment:
This wasn't working bc after the get request to '/logout' (a passport endpoint), the auth token is invalid and the user is effectively logged out of the front end.
BC of the routing protections in index.jsx this causes redirect to /login.
This line is kind of bogus bc when it runs, the connection to the store and router is inaccessible and history is undefined.
tl;dr: it wasn't doing anything except throwing a warning so I removed it.